src/main/java/ro/msg/learning/shop/dto/LocalDateTimeDTO.java

```diff
@@ -0,0 +1,50 @@
+package ro.msg.learning.shop.dto;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import ro.msg.learning.shop.entities.Address;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+@Data
+@Builder
+@NoArgsConstructor
+public class LocalDateTimeDTO {
+
+    private int year;
+    private int month;
+    private int dayOfMonth;
+    private int hour;
+    private int minute;
+    private int second;
+
+    public LocalDateTimeDTO(int year, int month, int dayOfMonth, int hour, int minute, int second) {
+        this.year = year;
+        this.month = month;
+        this.dayOfMonth = dayOfMonth;
+        this.hour = hour;
+        this.minute = minute;
+        this.second = second;
+    }
+
+    public LocalDateTime toEntity() {
+        return LocalDateTime.of(LocalDate.of(year, month, dayOfMonth), LocalTime.of(hour, minute, second));
```

Comment:
could you please consider using directly the appropriate of? for example 

of(int year, int month, int dayOfMonth, int hour, int minute, int second)
https://docs.oracle.com/javase/8/docs/api/java/time/LocalDateTime.html

Reply:
DONE: Using the appropiate "of()" method instead of the constructor.
The class LocalDateTimeDTO with it's fields helps for creating a proper LocalDateTime-**object/format** for the **POSTMAN**-test.